packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -227,3 +237,37 @@ export class RouterliciousDocumentServiceFactory implements IDocumentServiceFact
             this.snapshotTreeCache);
     }
 }
+
+/**
+ * Error returned by {@link RouterliciousDocumentServiceFactory.createContainer} when an error is thrown
+ * in {@link ITokenProvider.documentPostCreateCallback}.
+ * It is the consumer's responsibility to ensure that any state related to container creation is appropriately
+ * cleaned up in the event of failure.
+ * E.g. TODO
```

Comment:
Did you mean to include an example here?

Reply:
I was hoping to spark a conversation for this one. I don't know if we have any example scenarios we would like to call out here for the consumer's benefit or not. Do you have any suggestions?